lib/repo.js

```diff
@@ -71,8 +77,8 @@ function repo (options, callback) {
       }
 
       var urls = [
-        'https://api.github.com/repos/' + user + '/' + repo + '/contents/' + path + '.geojson',
-        'https://raw.github.com/' + user + '/' + repo + '/' + branch + '/' + path + '.geojson'
+        'https://api.github.com/repos/' + user + '/' + repo + '/contents/' + path + extension,
+        'https://raw.github.com/' + user + '/' + repo + '/' + branch + '/' + path + extension
       ]
```

Comment:
Yeah I think setting the default to `.geojson` is the right move. That way there's always a fallback but any other extension could be used too.